iceoryx_hoofs/source/posix_wrapper/thread.cpp

```diff
@@ -35,9 +35,10 @@ void setThreadName(iox_pthread_t thread, const ThreadName_t& name) noexcept
 
 ThreadName_t getThreadName(iox_pthread_t thread) noexcept
 {
-    char tempName[MAX_THREAD_NAME_LENGTH + 1U];
+    // NOLINTNEXTLINE(hicpp-avoid-c-arrays, cppcoreguidelines-avoid-c-arrays)
```

Comment:
Do we need a justification or todo for the C array suppression?

Reply:
Correct, but I added a `NOLINTJUSTIFICATION` line.